docs/Deploying-Wildfly.md

```diff
@@ -44,6 +46,7 @@ Similarly, Next Step contains the following configuration in `jboss-deployment-s
 ```
 
 The deployment descriptor requires configuration of the `com.wultra.powerauth.webflow.conf` and `com.wultra.powerauth.nextstep.conf` modules.
+Optionally the `com.wultra.powerauth.webflow-client.conf` module has to be configured also.
```

Comment:
Optionally and "has to" contradict themselves.

Reply:
Aaah, could be a little bit weird :). Changed.